ci-operator/jobs/openshift-psap/ci-artifacts/openshift-psap-ci-artifacts-main-presubmits.yaml

```diff
@@ -264,6 +264,172 @@ presubmits:
         secret:
           secretName: result-aggregator
     trigger: (?m)^/test( | .* )images,?($|\s.*)
+  - agent: kubernetes
+    always_run: true
```

Comment:
can you disable all the `always_run: true`?

Reply:
Thanks, done.